Class/Common/GeneralBuffs.cpp

```diff
@@ -197,6 +197,8 @@ QVector<ExternalBuffName> GeneralBuffs::get_buff_names_for_class(const QString&
 
     QVector<ExternalBuffName> common_raid_buffs = {
         ExternalBuffName::MarkOfTheWild,
+        ExternalBuffName::ArcaneIntellect,
+        ExternalBuffName::DivineSpirit,
```

Comment:
This will add these buffs for non-mana classes as well. I think it's OK for now, though.

Reply:
Yes, noticed the same. Good catch! Don't think it's much of an issue. The buff list just gets a bit long, but doesn't really hurt ;)